packages/from-to-location-picker/src/types.ts

```diff
@@ -0,0 +1,42 @@
+export type Location = {
+  lat: number;
+  lon: number;
+  name: string;
+  /**
+   * This is only used location that a user has saved. Can be either:
```

Comment:
First sentence does not make sense. Do you mean something like "This is a user-defined location type"?

Reply:
I copy pasted this from the flow types, but I agree something is wrong here. I've corrected it making as few changes as possible in 57e3522f13c3e0c11b4eef871ff4200451264d54